controls/radrichtextbox/document-elements/table/formatting-tables.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Formatting Tables
+page_title: Formatting Tables
+description: Formatting Tables
+slug: radrichtextbox-formatting-tables
+tags: table
+published: True
+position: 3
+---
+
+# Formatting Tables 
+
+__RadRichTextBox__ exposes an API that allows you to use various methods to style the tables. The methods exposed by the API can be wired to a UI and get executed upon user interaction with this UI. To learn more about the __Formatting API__ of the __RadRichTextBox__, read [this topic]({%slug radrichtextbox-features-formatting-api%}).
+
+All methods that allow you to interact with Tables are listed here: [Tables Properties and Methods]({%slug radrichtextbox-features-api%}).
+
+
+## Formatting the Table in Code
+
+The bellow code shows how you can create a table in the code behind and set its styles:
+
+#### __[C#] Generate a table and set its borders and background__
+
+{{region radrichtextbox-document-elements-formatting-tables_0}}   
+
+    public void SetBorders()
+    {
+        var table = new Table(10, 5);
+
+        var blackBorder = new Telerik.Windows.Documents.Model.Border(2, BorderStyle.Single, Colors.Green);
+        var grayBorder = new Telerik.Windows.Documents.Model.Border(3, BorderStyle.Single, Colors.Gray);
+        var grayBorder1 = new Telerik.Windows.Documents.Model.Border(1, BorderStyle.Single, Colors.Gray);
+
+        var rows = table.Rows.ToArray();
+
+        for (int row = 0; row < table.Rows.Count; row++)
+        {
+            var currentRow = rows[row];
+            currentRow.Height = 50;
+            var cells = currentRow.Cells.ToArray();
+
+            for (int col = 0; col < currentRow.Cells.Count; col++)
+            {
+                var currnetCell = cells[col];
+                currnetCell.TextAlignment = Telerik.Windows.Documents.Layout.RadTextAlignment.Center;
+                currnetCell.VerticalAlignment = Telerik.Windows.Documents.Layout.RadVerticalAlignment.Center;
+
+                if (row == 0)
+                {
+                    currnetCell.Borders = new TableCellBorders(blackBorder, grayBorder1, blackBorder, grayBorder);
+                }
+                else
+                {
+                    currnetCell.Borders = new TableCellBorders(blackBorder, grayBorder1, blackBorder, grayBorder1);
+                }
+
+                if (col % 2 == 0)
+                {
+                    currnetCell.Background = Color.FromArgb(117, 220, 220, 220);
+                }
+
+            }
+        }
+
+        SetTableData(table);
+
+        var editor = new RadDocumentEditor(this.radRichTextBox.Document);
+        editor.InsertTable(table);
+    }
+
+{{endregion}}
+
+#### __[VB.NET] Generate a table and set its borders and background__
```

Comment:
Add numbering to the caption:
__[C#] Example 1: Generate a table and set its borders and background__